### PR TITLE
[Doc Improvement][Retrieve Token Azure app][3189121]

### PR DESCRIPTION
**Pull request by v-bvishnu**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/37c6f7d08185536694780149993408afd9118925/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 34 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1720fbd4-e895-471c-98e5-6ca1456d4803) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/dc564b5bdbbd0c13942aa70ec3fe0af5a2bdcc8c/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/64df26d2-bb93-4d21-bf3f-dd32ee5b4953) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/cb67aed70c23ab69865687a5276d4bc4621a1beb/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 31 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5dfb138a-6ea1-4cfa-86ba-0be17ccde269) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/eef82651a4be679ce8c3aa11e3fde93c8fb6680c/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/80bc5b29-3163-40ae-a4c2-c54dde45729d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/3eaf33cc60c955b9828bcb064bacba73646f6ad1/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/71f7c40f-e519-4575-91db-a5b0a4a7442c) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/328818d275c6b0949b38933d6ab04896a1e1097e/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/8d7e4a16-610d-4f5e-ae77-8913c6659936) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by Wajeed-msft — APPROVED** [review]

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/307eee384ac6709b1a9be131b8f7578065812f1b/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6ab03492-a4d0-4045-9623-50954e2dd72d) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/d102c2724e5b7928f2674a07380fca7bc367f8a1/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/54e164ec-8716-4b9d-81e7-d79e651f2cff) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md [hunk 1]
```diff
@@ -116,7 +116,7 @@ You can configure authentication for multiple platforms as long as the URL is un
 
 You need to acquire an access token for Microsoft Graph. You can do so by using Microsoft Entra on-behalf-of (OBO) flow.
 
-The current implementation for single sign-on (SSO) is limited to user-level permissions, which aren't usable for making Graph calls. To get the permissions and scopes needed to make a Graph call, SSO apps must implement a custom web service to exchange the token received from the Teams JavaScript library for a token that includes the needed scopes. You can use Microsoft Authentication Library (MSAL) for fetching the token from the client side.
+The current implementation for SSO is limited to user-level permissions, which aren't usable for making Graph calls. To get the permissions and scopes needed to make a Graph call, SSO apps must implement a custom web service to exchange the token received from the Teams JavaScript library for a token that includes the needed scopes. You can use Microsoft Authentication Library (MSAL) for fetching the token from the client side.
```
Comment (TheBluLion) [inline msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md:119]: SSO should reamain single sign-on (SSO) to remain consistent with format above and to provide clarity on the acronym.

File: msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md
Comment (TheBluLion) [inline msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md:1]: On line 21, we should make a stronger recommendation that less permissive scopes (delegated) should be preferred over application permissions

File: msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md [hunk 2]
```diff
@@ -204,7 +204,7 @@ If you need to access Microsoft Graph data, configure your server-side code to:
 > [!IMPORTANT]
 >
 > - As a best practice for security, always use [server-side code to make Microsoft Graph calls](/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow#middle-tier-access-token-request) or other calls that require passing an access token. This helps protect the token from being intercepted or leaked. DO NOT return the OBO token to the client because it would then enable the client to make direct calls to Microsoft Graph.
->
+> - If you've two separate apps registered in Microsoft Entra ID and require individual tokens for each app, use the [OBO flow](/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) to enable communication between the apps.
```
Comment (TheBluLion) [inline msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md:207]: Two separate apps registered in Microsoft Entra ID will require individual tokens for each app. Use the [OBO flow](/azure/active-directory/develop/v2-oauth2-on-behalf-of-flow) to enable communication between the apps.

Comment (v-bvishnu) [reply msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md:207]: Updated.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/55fe05f042ef15ceffb5765788dacaa1b498938a/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 89 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/53827c37-c5f2-4a49-92df-9c904c0a7203) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

File: msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md (hunk 1 above)
Comment (v-bvishnu) [reply msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md:119]: Updated.

File: msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md
Comment (v-bvishnu) [reply msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md:1]: Updated as: We recommend using delegated permissions for the signed-in user. If your application doesn't need a signed-in user, consider using application permissions, also known as the app-only access scenario. Only administrators can grant consent for application permissions. For more information, see [application permissions](/graph/permissions-overview#application-permissions).

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards 
 
**We currently enforce a minimum score of 80.**
 
Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology:
| Article | Score | Issues | Scorecard | Processed |
| ------- | ----- | ------ | --------- | --------- |
| [msteams-platform/tabs/how-to/authentication/tab-sso-graph-api.md](https://github.com/MicrosoftDocs/msteams-docs/blob/6565e8bddbb7f0f2c03cd11c4d9844dc3a64f136/msteams-platform%2Ftabs%2Fhow-to%2Fauthentication%2Ftab-sso-graph-api.md) | 90 | 30 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/00b31875-3f8e-47ee-b15f-814a5906d590) | [:white_check_mark:](# "Successfully processed") |

**More info about Acrolinx**
 
Use the Acrolinx extension, or sidebar, in Visual Studio Code to check spelling, grammar, style, tone, clarity, and key terminology when you're creating or updating content. For more information, see [Use the Visual Studio Code extension to run Acrolinx locally](https://review.docs.microsoft.com/office-authoring-guide/acrolinx-vscode).

**Review by TheBluLion — APPROVED** [review]

**Review by v-deabira — APPROVED** [review]

**Pull request merged**